### PR TITLE
[15.0][MIG] account_move_substate

### PR DESCRIPTION
**Pull request by bosd**
standard migration

### CODE REVIEW

**Comment by bosd** [comment]
@Saran440 Can you please review this one as well?

**Comment by bosd** [comment]
@Draynyel-bosd Can you please review?

**Comment by Saran440** [comment]
@bosd I think we should review v14 first https://github.com/OCA/account-invoicing/pull/1001
and after merged, you can rebase that you commit for improved code to v15
Selection_004

**Comment by bosd** [comment]
> @bosd I think we should review v14 first #1001

Actually, that one is superseeded by #1594.
In the ideal world, yes, we would start on V14 and forward migrate it to 17.
However, client needed it on V16. So V16 is/was the priority.

The other versions / PR are bonus. added them, in case somebody wants to merge them.

**Review by Dranyel-Bosd — APPROVED** [review]
Functional review :+1:

**Review by bofiltd — APPROVED** [review]
LGTM

**Comment by bosd** [comment]
@etobella V16 is already merged. Can you please merge this one as well?

File: account_move_substate/readme/DESCRIPTION.rst
Comment (etobella) [inline account_move_substate/readme/DESCRIPTION.rst]: `With this module`

**Review by etobella — COMMENTED** [review]
I am unable to test it using README on runboat. Can you review the readme please?

**Comment by bosd** [comment]
@etobella The description is updated. I think this one is ready now.

File: account_move_substate/readme/USAGE.rst
Comment (etobella) [inline account_move_substate/readme/USAGE.rst]: It is not under database structure

**Review by etobella — COMMENTED** [review]
@bosd, I don't see the documentation commit :thinking: 

We also found a nitpicking documentation mistake.

**Comment by bosd** [comment]
> I don't see the documentation commit 🤔

it's here https://github.com/OCA/account-invoicing/pull/1595/commits/e9bc9d40c9d68f2122f3d793724b6e2cbc845b8b

File: account_move_substate/readme/USAGE.rst
Comment (bosd) [reply account_move_substate/readme/USAGE.rst]: Good catch. It's actually under Settings > Technical > Sub State Configuration and add a "Base Substate".

We actually misspelled `Sub State` in multiple places (in base module) as the correct spelling is `Substate`.
Source: https://dictionary.cambridge.org/dictionary/english/sub-state

edit: will commit these changes, when I am at my dev machine.

**Comment by bosd** [comment]
@etobella is it ok now?

**Pull request merged**
